tutorials/Image_And_Icons_Manipulations/README.md

```diff
@@ -337,3 +337,34 @@ fun asImageAsset(image: BufferedImage): ImageAsset {
 ```
 
 ![Tray icon](tray_icon.png)
+
+## Loading XML vector images

```

Comment:
## Loading XML vector images
Compose for Desktop supports XML vector images.
XML vector images come from the world of [Android](https://developer.android.com/guide/topics/graphics/vector-drawable-resources).
We implemented it on the desktop so we can use common resources in a cross-platform application.

SVG files can be converted to XML with [Android Studio](https://developer.android.com/studio/write/vector-asset-studio#svg) or with [third-party tools](https://www.google.com/search?q=svg+to+xml).
Suppose we have an XML image placed in the `resources/images` directory in our project.

Reply:
is it
Compose for Desktop supports XML vector images.

or

Compose for Desktop supports XML vector graphics.

Reply:
Thanks!

> Loading XML vector images
Fixed.

> Compose for Desktop supports XML vector images.
I think "XML vector images" is a more suitable name.